include/net/lwm2m_client_utils.h

```diff
@@ -124,9 +124,11 @@ int lwm2m_update_connmon(void);
 
 #if defined(CONFIG_LWM2M_CLIENT_UTILS_SIGNAL_MEAS_INFO_OBJ_SUPPORT)
 #define ECID_SIGNAL_MEASUREMENT_INFO_OBJECT_ID 10256
+#include "modem/lte_lc.h"
```

Comment:
I'd prefer to see this among other includes (doesn't really need to be conditional).